runtime/lua/nvim/health.lua

```diff
@@ -383,13 +383,27 @@ local function check_terminal()
   end
 end
 
+local function check_external_tools()
+  health.start('External Tools')
```

Comment:
Should be in "Performance" section unless @clason strongly objects

Reply:
Not strongly, but I would like to extend this section to other "optional tools" (like `curl`).